spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe VideosController do
+  describe "GET show" do
+
+    context "with authenticated users" do 
+      before do
+        #user = User.create(email: 'user@example.com', password: 'password', full_name: "Example" )
+        session[:user_id] = Fabricate(:user).id #user.id
+      end
+
+    	it "sets @video instance variable" do
+        video = Fabricate(:video)#Video.create(title: "Monk", description: "Monk is cool.")
+        get :show, id: video.id
+        expect(assigns(:video)).to eq(video)
+      end
+    end
+
+    it "sets @reviews instance variabe for authenticated users" do
+      session[:user_id] = Fabricate(:user).id #user.id
+      video = Fabricate(:video)
+      review1 = Fabricate(:review, video: video)
+      review2 = Fabricate(:review, video: video)
+      get :show, id: video.id
+      assigns(:reviews).should =~ [review1, review2]
```

Comment:
The new syntax is `expect(assigns(:reviews)).to match_array([review_1, review_2])`.